test/lib/ex_aws/config_test.exs

```diff
@@ -9,4 +9,17 @@ defmodule ExAws.ConfigTest do
     |> Map.get(:config)
     |> Map.get(:access_key_id) == value
   end
+
+  test "security_token is configured properly" do
+    value = "security_token"
+    System.put_env("AWS_SECURITY_TOKEN", value)
+    assert %{
+      __struct__: %{ config_root: [access_key_id: {:system, "AWS_SECURITY_TOKEN"}]},
```

Comment:
What's going on here? Why are we not following elixir conventions on how `__struct__` is used?